### PR TITLE
Support 0.56.0

### PR DESCRIPTION
**Pull request by ikesyo**
Three patches are not needed anymore:

- https://github.com/facebook/react-native/commit/cf036dbc7af16a8453c115372694dc51e8086fcf
- https://github.com/facebook/react-native/commit/c09d509c2b8a5a02701829e1f0ace8081ce64277
- https://github.com/facebook/react-native/commit/ca515e99a39dcadbbfb653205e2929dec7e7096b

### CODE REVIEW

File: lib/cocoapods-fix-react-native/versions/0_56_0-rc_5-post.rb [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  end
+  code = File.read(file)
+  if code.include?(old_code)
+    Pod::UI.message "Patching #{file}", '- '
+    FileUtils.chmod('+w', file)
+    File.write(file, code.sub(old_code, new_code))
+  end
+end
+
+def fix_unused_yoga_headers
```
Comment (ikesyo) [inline lib/cocoapods-fix-react-native/versions/0_56_0-rc_5-post.rb]: @orta Looks like https://github.com/facebook/react-native/commit/cf036dbc7af16a8453c115372694dc51e8086fcf will make this patch unnecessary, right?

Comment (orta) [reply lib/cocoapods-fix-react-native/versions/0_56_0-rc_5-post.rb]: Yes, I think so 👍

Comment (ikesyo) [reply lib/cocoapods-fix-react-native/versions/0_56_0-rc_5-post.rb]: Thanks! I'll remove this as well.

**Comment by orta** [comment]
Perfect, yeah, this looks good to me - thanks!

**Comment by ikesyo** [comment]
Looks like the new version is shipped: https://rubygems.org/gems/cocoapods-fix-react-native

**Pull request merged**